samples/star-wars-api/Helpers.fs

```diff
@@ -22,16 +25,25 @@ module StringHelpers =
 
     let isNullOrWhiteSpace (str : string) = String.IsNullOrWhiteSpace (str)
 
-    
+
+[<AutoOpen>]
+module LoggingHelpers =
+
     open Microsoft.Extensions.DependencyInjection
     open Microsoft.Extensions.Logging
 
     type IServiceProvider with
-        member sericeProvider.CreateLogger (``type`` : Type) =
-            let loggerFactory = sericeProvider.GetRequiredService<ILoggerFactory>()
+        member sProvider.CreateLogger (``type`` : Type) =
+            let loggerFactory = sProvider.GetRequiredService<ILoggerFactory>()
```

Comment:
let's keep it as it was

Reply:
I don't understand why?  To me, `sericeProvider` looks like an accidental typo.

Reply:
Yes, `serviceProvider`. I just missed the type, sorry

Reply:
Fixed in 4dcbd4ca80b034738b950e4909ed15ff27f14ad1 .